cloud/scope/powervs_cluster.go

```diff
@@ -1866,8 +1869,12 @@ func (s *PowerVSClusterScope) DeleteLoadBalancer() (bool, error) {
 		if _, err = s.IBMVPCClient.DeleteLoadBalancer(&vpcv1.DeleteLoadBalancerOptions{
 			ID: lb.ID,
 		}); err != nil {
-			s.Error(err, "error deleting the load balancer")
-			return false, err
+			errs = append(errs, fmt.Errorf("error deleting the load balancer: %w", err))
+			continue
+		}
+
+		if len(errs) > 0 {
+			return false, kerrors.NewAggregate(errs)
 		}
 		return true, nil
```

Comment:
If you return here won't it block the deletion of remaining items until it succeeds?

Reply:
That's right, changed it. PTAL!